Makefile

```diff
@@ -26,9 +26,18 @@ publish: build
 	git tag "v$(MIG_VERSION)"
 	git push origin main "v$(MIG_VERSION)"
 
+# unit tests
 test: build
 	go test -v ./...
 
+# integration tests
+#integration: build
+#	cd tests/postgres
```

Comment:
This stuff doesn't work at all. Also, `pushd` isn't available. Probably need to update test.mjs to change working directory.